{22624: 0}
{22624: 0.4421137887984514}
{22624: 0.45640680007636547}
{22624: 0.4608111041598022}
{22624: 0.8647052231244743}
{22624: 1.0660454230383039}
{22624: 1.6581450668163598}
{22624: 1.9471627157181501}
{22624: 2.5210156054235995}
{22624: 2.7800183980725706}
{22624: 3.3551425668410957}
{22624: 3.656695469748229}
{22624: 4.178161643911153}
{22624: 4.413791185244918}
{22624: 4.974999677855521}
{22624: 5.247638711240143}
{22624: 5.786744688171893}
{22624: 6.16243414208293}
{22624: 6.166756258811802}
{22624: 6.662658306770027}
{22624: 7.272619119845331}
{22624: 8147.929332087282}
{22624: 8149.945056621451}
{22624: 10347.467419668566}
{22624: 27819.706391219515}
{22624: 27819.934768500272}
{22624: 27819.938533338252}
{22624: 27820.72477908898}
{22624: 27820.993561640847}
{22624: 27823.9246212421}
Traceback (most recent call last):
  File "ra.py", line 41, in <module>
    vm.verify_full_state()
  File "/ssd//tools/git/RA/vm.py", line 285, in verify_full_state
    self.run()
  File "/ssd//tools/git/RA/vm.py", line 845, in run
    ret = func(s, *funcarg)
  File "/ssd//tools/git/RA/vm.py", line 1438, in op_calldataload
    raise DevelopmentErorr()
exceptions.DevelopmentErorr: Implementation or design error. 
